pkg/manager/manager.go

```diff
@@ -219,6 +219,7 @@ type Options struct {
 
 	// HealthProbeBindAddress is the TCP address that the controller should bind to
 	// for serving health probes
+	// It can be set to "0" or "" to disable the health serving.
```

Comment:
```suggestion
	// It can be set to "0" or "" to disable serving the health probe.
```
nit

Can you please add the same comment in: https://github.com/kubernetes-sigs/controller-runtime/blob/c48baad70c539a2efb8dfe8850434ecc721c1ee1/pkg/config/v1alpha1/types.go#L113

Reply:
already added